modules/virt-configuring-vm-real-time.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-cluster-realtime-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-vm-real-time_{context}"]
+= Configuring a virtual machine for real-time workloads
+
+You can configure a virtual machines (VM) to run real-time workloads.
+
+.Prerequisites
+* Your cluster is configured to run real-time workloads.
+* You have installed the `virtctl` tool.
+
+.Procedure
+. Create a `VirtualMachine` manifest to include information about CPU topology, CRI-O annotations, and huge pages:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: realtime-vm
+spec:
+  template:
+    metadata:
+      annotations:
+        cpu-load-balancing.crio.io: disable <1>
+        cpu-quota.crio.io: disable <2>
+        irq-load-balancing.crio.io: disable <3>
+    spec:
+      domain:
+        cpu:
+          dedicatedCpuPlacement: true
+          isolateEmulatorThread: true
+          model: host-passthrough
+          numa:
+            guestMappingPassthrough: {}
+          realtime: {}
+          sockets: 1 <4>
+          cores: 4 <5>
+          threads: 1
+        devices:
+          autoattachGraphicsDevice: false
+          autoattachMemBalloon: false
+          autoattachSerialConsole: true
+        ioThreadsPolicy: auto
+        memory:
+          guest: 4Gi
+          hugepages:
+            pageSize: 1Gi <6>
+        terminationGracePeriodSeconds: 0
+# ...
+----
+<1> This annotation specifies that load balancing is disabled for CPUs that are used by the container.
+<2> This annotation specifies that the CPU quota is disabled for CPUs that are used by the container.
+<3> This annotation specifies that interrupt request (IRQ) load balancing is disabled for CPUs that are used by the container.
+<4> The number of sockets inside the VM. 
+<5> The number of cores inside the VM. This must be a value greater than or equal to `1`.
+<6> The size of the huge pages. The possible values for x86-64 architectures are `1Gi` and `2Mi`. In this example, the request is for 4 huge pages of size 1 Gi.
+
+. Apply the `VirtualMachine` manifest:
++
+[source,terminal]
+----
+$ oc apply -f <file_name>.yaml
+----
+
+. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 8 operating system:
+.. Run the following command to connect to the VM console:
++
+[source,terminal]
+----
+$ virtctl console <vm_name>
+----
+
+.. Configure huge pages by using the GRUB boot loader command-line interface. In the following example, 8 1-G huge pages are specified.
```

Comment:
> 1-G huge pages

Are you sure that using the hyphen is a valid tech syntax? I'm asking because I have never encountered this before, so I apologize if I'm just making unnecessary noise here.

Reply:
Removed the hyphen